MOBILE_CLIENT/COMMON/_HIGH/OAUTH_SCHEME_ACCOUNT_TAKEOVER/recommendation.md

```diff
@@ -0,0 +1,147 @@
+To address the vulnerability, it is recommended to not use the custom scheme to redirect authentication tokens.
+
+Developers should instead consider one of the following options:
+
+- App to app integration like Google Identity Services and Facebook Express Login for Android
+- Android's verifiable AppLinks  
+- iOS associated domains.
```

Comment:
These should be grouped.

Reply:
I think it's better to keep them separate for more emphasis